tensorflow/tools/pip_package/setup.py

```diff
@@ -63,7 +63,7 @@
     'numpy >= 1.16.0, < 1.19.0',
     'opt_einsum >= 2.3.2',
     'protobuf >= 3.9.2',
-    'tensorboard >= 2.2.0, < 2.3.0',
+    'tensorboard >= 2.3.0, < 3',
```

Comment:
Should the upper bound be `2.4.0` since these packages are in sync?

Reply:
Good question, this is an intentional change, starting with 2.3.  The TensorBoard team has opted with this more relaxed dependency as part of a plan to release user-updates more frequently.  The minor version piece is decoupled, while the major version is still in sync.

(btw I realize the piper origin revId in the description is incorrect, just updated)
I'll also update the PR/commit description so that it's clearer.

cc @nfelt